### ISSUE TITLE
Feature Request - Nested List

### ISSUE THREAD

**Issue by swarnendumaiti** [maltaisn/another-notes-app#28]
Please bring in the ability to make Nested List like shown on this image. It helps to check multiple sublists in a to-do at once and also creates much better category classifications in the list.

![Screenshot_20210704-113702~2](https://user-images.githubusercontent.com/86847643/124380207-1992d800-dcd9-11eb-8fe8-1faacde3b17a.png)

**Comment by swarnendumaiti** [comment]
Another small request although not a necessary one. 
- Pressing Backspace Button in lists will let the user remove the list and go to previous list. Further Backspace will lead to delete that list too and so on. Currently there's a cross button to remove a list, but using the Backspace button might come in handy.

This is obviously a subtle change, but do not implement it, if it makes your code complex.

**Comment by maltaisn** [comment]
Backspace should work, but it has beem known to not work with some keyboards. What keyboard are you using?

**Comment by swarnendumaiti** [comment]
I'm using OpenBoard : https://github.com/dslul/openboard

If my keyboard's the issue, then kindly let me know. I'll delete my comment.

**Comment by swarnendumaiti** [comment]
Yeah it seems my keyboard's the issue. I used Gboard, and it works fine with Backspace.

**Comment by maltaisn** [comment]
See https://github.com/dslul/openboard/issues/331. Although backspace does work with OpenBoard in Google Keep Notes so there's definitely something to do about it. I'll look into it eventually.

**Comment by swarnendumaiti** [comment]
Yes, it works on Google Keep, and so I'm so habituated with the action. Now that I've left Keep Notes for a more private alternative, I feel it missing. Please do look into it.
